src/magic.cpp

```diff
@@ -909,13 +909,14 @@ std::string spell::enumerate_targets() const
         return all_valid_targets[0];
     }
     std::string ret;
+    // @todo if only we had a function to enumerate strings and concatenate them...
```

Comment:
`enumerate_as_string` in `output.h`?

Reply:
Yes, I was ironically referring to that one. As of now, I leave it to someone else to actually implement it.